samples/subsys/shell/shell_module/boards/mec15xxevb_assy6853.overlay

```diff
@@ -0,0 +1,6 @@
+/ {
```

Comment:
Is zephyr,console needed again in the overlay?  zephyr,console is already part of board dts.

https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/mec15xxevb_assy6853/mec15xxevb_assy6853.dts#L17